specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json

```diff
@@ -694,6 +705,11 @@
     "ConnectedClusterPatchProperties": {
       "type": "object",
       "properties": {
+        "privateLinkEnabled": {
+          "type": "boolean",
+          "default": false,
```

Comment:
>"default": false, [](start = 10, length = 17)

Should there be a default in patch?

Reply:
It should not. Thanks for pointing this out